src/domain/event/eventInfo/__tests__/EventInfo.test.tsx

```diff
@@ -364,3 +391,225 @@ describe('OrganizationInfo', () => {
+
+  it('should should not render super event title when super event is not given', async () => {
+    render(<EventInfo event={event} eventType="event" />, {
+      mocks,
+    });
+    await actWait();
+
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.superEvent.title,
+      })
+    ).not.toBeInTheDocument();
+  });
+});
+
+describe('subEvents', () => {
+  const meta: Meta = {
+    count: 20,
+    next:
+      // eslint-disable-next-line max-len
+      'https://api.hel.fi/linkedevents/v1/event/?include=keyword,location&page=2&sort=start_time&start=2020-08-11T03&super_event=hel:123',
+    previous: null,
+    __typename: 'Meta',
+  };
+
+  const subEventsResponse = {
+    ...fakeEvents(
+      10,
+      range(1, 11).map((i) => ({
+        endTime: addDays(new Date(endTime), i).toISOString(),
+        startTime: addDays(new Date(startTime), i).toISOString(),
+        typeId: i % 2 === 0 ? EventTypeId.Course : EventTypeId.General,
+        superEvent: { internalId: superEventInternalId },
+      }))
+    ),
+    meta,
+  };
+
+  const subEventsLoadMoreResponse = {
+    ...fakeEvents(
+      10,
+      range(11, 21).map((i) => ({
+        endTime: addDays(new Date(endTime), i).toISOString(),
+        startTime: addDays(new Date(startTime), i).toISOString(),
+        superEvent: { internalId: superEventInternalId },
+      }))
+    ),
+    meta: { ...meta, next: null },
+  };
+
+  const getSubEventsMocks = ({
+    eventType = 'event',
+    response,
+    variables,
+  }: {
+    eventType: EventType;
+    response: EventListResponse;
+    variables?: EventListQueryVariables;
+  }) =>
+    createOtherEventTimesRequestAndResultMocks({
+      superEventId: event.id,
+      response,
+      variables,
+      type: eventType,
+    });
+
+  const firstSubEventsLoadMock = getSubEventsMocks({
+    eventType: 'event',
+    response: subEventsResponse,
+  });
+
+  const secondSubEventsLoadMock = getSubEventsMocks({
+    variables: { page: 2 },
+    response: subEventsLoadMoreResponse,
+    eventType: 'event',
+  });
+
+  const mocksWithSubEvents = [
+    ...mocks,
+    firstSubEventsLoadMock,
+    secondSubEventsLoadMock,
+  ];
+
+  it('should render sub events title and content when sub events are given', async () => {
+    render(<EventInfo event={event} eventType="event" />, {
+      mocks: mocksWithSubEvents,
+    });
+    await actWait();
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.subEvents.title,
+      })
+    ).toBeInTheDocument();
+    await testSubEvents();
+  });
+
+  it.each([
+    [EventTypeId.General, '/fi/events/'],
+    [EventTypeId.Course, '/fi/courses/'],
+  ])(
+    'should navigate to sub events page when it is clicked',
+    async (eventTypeId: EventTypeId, url: string) => {
+      const { history } = render(
+        <EventInfo event={event} eventType="event" />,
+        {
+          mocks: mocksWithSubEvents,
+        }
+      );
+      await actWait();
```

Comment:
Here we could replace actWait with findBy. I think it is clearer what we are waiting :)
`const eventsList = await screen.findByTestId(subEventsListTestId);`